web/tests/stream_pill.test.js

```diff
@@ -32,16 +32,24 @@ peer_data.set_subscribers(denmark.stream_id, [1, 2, 3, 77]);
 peer_data.set_subscribers(sweden.stream_id, [1, 2, 3, 4, 5]);
 
 const denmark_pill = {
-    stream_name: denmark.name,
-    stream_id: denmark.stream_id,
     type: "stream",
-    display_value: "#Denmark: 3 users",
+    display_value: "Denmark: 3 users",
+    stream: {
+        name: denmark.name,
+        id: denmark.stream_id,
+        invite_only: undefined,
+        is_web_public: undefined,
```

Comment:
Why are these values `undefined`? I would think you'd want true/false for them.

Reply:
These properties are coming `undefined` from the `StreamSubscription`. So to match that had to set it to `undefined`.

Fixed it by using if conditions.